SPECS/cloud-init/cloud-init.spec

```diff
@@ -141,7 +143,10 @@ make check %{?_smp_mflags}
 %config(noreplace) %{_sysconfdir}/cloud/cloud.cfg.d/10-azure-kvp.cfg
 
 %changelog
-* Fri Jun 03 2022 Chris Patterson <cpatterson@microsoft.com> - 22.2-1
+* Wed Jun 08 2022 Tom Fay <tomfay@microsoft.com> - 22.2-3
+- Add missing e2fsprogs dependency
+
+* Fri Jun 03 2022 Chris Patterson <cpatterson@microsoft.com> - 22.2-2
```

Comment:
https://github.com/microsoft/CBL-Mariner/pull/3104 didn't reset the release to 1, so I've bumped from 2 to 3.